src/chrome/content/rules/MeiTuan.com.xml

```diff
@@ -0,0 +1,75 @@
+<!--
+	Related:
+		MeiTuan.net.xml
+
+	Cities:
+		There are more and more cities: https://www.meituan.com/index/changecity/initiative
+		While this ruleset be created on 28 May 2016, there were 886 cites.
+		On 20 Feb 2017, the number is 964.
+		We get error from travis: “The job exceeded the maximum time limit for jobs, and has been terminated.”
+		To fix this issue, maybe exclusion is the best choise.
+
+	Different http/https content:
+		^meituan.com
+		tech.meituan.com
+		union.meituan.com
+		zhao.meituan.com
+
+	Invalid cert:
+		ck.dsp.meituan.com
+		api.mobile.meituan.com
+
+	MCB:
+		i.meituan.com/mobile/
+		waimai.meituan.com
+-->
+
+<ruleset name="MeiTuan.com">
+	<!--Directly:-->
+	<target host="www.meituan.com"/>
+	<target host="analytics.meituan.com"/>
+	<target host="b.meituan.com"/>
+	<target host="daili.meituan.com"/>
+	<target host="hotel.meituan.com"/>
+	<target host="mos.meituan.com"/>
+	<target host="p0.meituan.com"/>
+	<target host="p1.meituan.com"/>
+	<target host="passport.meituan.com"/>
+	<target host="report.meituan.com"/>
+	<target host="s0.meituan.com"/>
+	<target host="s1.meituan.com"/>
+		<test url="http://s1.meituan.com/bs/js/?f=mta-js:mta.min.js" />
+	<target host="kaidian.waimai.meituan.com"/>
+		<test url="http://kaidian.waimai.meituan.com/images/welcome/step1.png" />
+	<target host="waimaie.meituan.com"/>
+
+	<!--MCB:-->
+	<target host="i.meituan.com"/>
+		<exclusion pattern="^http://i\.meituan\.com/mobile/" />
+		<test url="http://i.meituan.com/mobile/down/meituan" />
+
+	<target host="waimai.meituan.com"/>
+		<exclusion pattern="^http://waimai\.meituan\.com/(?!static/)" />
+		<test url="http://waimai.meituan.com/static/img/logos/small_3.png" />
+		<test url="http://waimai.meituan.com/mobile/download/default" />
+
+	<!--For Cities:-->
+	<target host="*.meituan.com"/>
+		<test url="http://bj.meituan.com/" />
+		<test url="http://sh.meituan.com/" />
+		<test url="http://gz.meituan.com/" />
+		<test url="http://sz.meituan.com/" />
+
+	<exclusion pattern="^http://tech\.meituan\.com/" />
+		<test url="http://tech.meituan.com/" />
+	<exclusion pattern="^http://union\.meituan\.com/" />
+		<test url="http://union.meituan.com/" />
+	<exclusion pattern="^http://zhao\.meituan\.com/" />
+		<test url="http://zhao.meituan.com/" />
+	<exclusion pattern="^http://ck\.dsp\.meituan\.com/" />
+		<test url="http://ck.dsp.meituan.com/beacon?adx=tanx" />
+	<exclusion pattern="^http://api\.mobile\.meituan\.com/" />
+		<test url="http://api.mobile.meituan.com/appupdate/download/simple/waimai?channel=1013" />
+
+	<rule from="^http:" to="https:" />
```

Comment:
Maybe you can use a more strict regex like `^http://(\w+)\.meituan\.com/` to avoid redirects on 4th level subdomains (*.*.meituan.com)